# CDRF-CA_Dairy_Fecal_Microbiome
Scripts for sequence processing and R analysis 
16s amplicon sequencing of fecal samples from 150 cows on 15 different farms across california commerical diaries.  

1. For Submission to NCBI is a file that describes the demultiplexing of raw sequences and prepping them for NCBI submission.
2. R_session_info is the output of R session to show all packages used and their version.
3. Scripts_for_processing contains filtering steps (phyloseq), beta & alpha diversity and differential abundance testing (DESeq2). Figures 1-5 and Supplemental tables 1-3.
4. Sequence coverage file are the scripts to describe the coverage of sequencing for this data set.
5. Controls file are the scripts to show taxa in control samples.
6. MaSaLin Analysis is the R scripts used to run and generate output from MaSaLin (Fig 6 & 7, Supplemental Table 4).
7. Pathogen_Tree are qiime and other bash scripts used to make a reference tree of full lenght 16S rRNA sequences from RDP database.

Paper:
